docs/content/Getting-Started-Docker.md

```diff
@@ -29,26 +26,48 @@ services:
       # 3000 is a port for Playground web server
       # it is available only in dev mode
       - 3000:3000
-    env_file: .env
     volumes:
       - ./schema:/cube/conf/schema
 ```
 
 ## 2. Configure Cube.js
 
+<!-- prettier-ignore-start -->
+[[info |]]
+| This step assumes you can connect to a database instance. If you're unable
+| to connect to a remote instance, please use a Docker image to run one in
+| your local development environment.
+<!-- prettier-ignore-end -->
+
+### Developer Playground
+
+The [Developer Playground][ref-devtools-playground] has a database connection
```

Comment:
Should we put a note here saying: you can skip the rest of the steps (or at least the one on env variables) if you're configuring in Developer Playground?

Reply:
Yep, great catch 😀